README.md

```diff
@@ -93,57 +93,126 @@ dicom-anonymizer InputFilePath OutputFilePath --dictionary dictionary.json
 ```
 
 
-## Custom actions from dictionary file
+## Custom/overrides actions
 
-If you use the file anonymizer.py, then you can add custom actions:
-```json
-{
-    "(0x0002, 0x0002)": "ActionNameNotReferenced",
-    "(0x0003, 0x0003)": "ActionNameNotReferenced",
-    "(0x0004, 0x0004)": "ActionNameNotReferenced1",
-    "(0x0005, 0x0005)": "ActionNameNotReferenced2"
-}
-```
-Then, modify your file anonymizerUser.py :
+Here is a small example which keep all metadata but update the series description
+by adding a suffix passed as a parameter.
 ```python
-import dicomanonymizer.anonymizer
-
-# Functions need to have in input a dataset and a tag
-def ActionNameNotReferenced(dataset, tag):
-    # Extract the element from the dataset :
-    element = dataset.get(tag)
-    # You can update the value as here :
-    element.value = ''
-	print('custom action')
-def ActionNameNotReferenced1(dataset, tag):
-	print('custom action 1')
-def ActionNameNotReferenced2(dataset, tag):
-	print('custom action 2')
-
-actionsTagMap = {
-	(0x0001, 0x0001): "keep", #will use the default method in the dicom anonymizer
-	(0x0002, 0x0002): ActionNameNotReferenced,
-	(0x0003, 0x0003): ActionNameNotReferenced2
-}
-
-# Generate a custom dictionary that will be override the default one
-actionsMap = dicomanonymizer.anonymizer.generateActionsDictionary(actionsTagMap)
-
-# Send the map to the main function
-dicomanonymizer.anonymizer.anonymize(actionsMap)
+import argparse
+from dicomanonymizer import *
+
+def main():
+    parser = argparse.ArgumentParser(add_help=True)
+    parser.add_argument('input', help='Path to the input dicom file or input directory which contains dicom files')
+    parser.add_argument('output', help='Path to the output dicom file or output directory which will contains dicom files')
+    parser.add_argument('--suffix', action='store', help='Suffix that will be added at the end of series description')
+    args = parser.parse_args()
+
+    InputPath = args.input
+    OutputPath = args.output
+
+    dictionary = {}
```

Comment:
I think dictionary is not a good name for this variable (it is not a good idea to mention the data structure in the variable name but instead the role of this variable).

`extraAnonymizationRules` or something like that seems more explicit.